Modules/_datetimemodule.c

```diff
@@ -1832,11 +1832,18 @@ wrap_strftime(PyObject *object, PyObject *format, PyObject *timetuple,
               PyObject *tzinfoarg)
 {
     PyObject *result = NULL;            /* guilty until proved innocent */
+    PyObject *strftime = NULL;          /* time.strftime */
 
     PyObject *zreplacement = NULL;      /* py string, replacement for %z */
     PyObject *colonzreplacement = NULL; /* py string, replacement for %:z */
     PyObject *Zreplacement = NULL;      /* py string, replacement for %Z */
     PyObject *freplacement = NULL;      /* py string, replacement for %f */
+#ifdef NORMALIZE_CENTURY
+    PyObject *year_str;                     /* py string, year */
+    PyObject *year;                         /* py int, year */
+    long year_long;                         /* year as long int */
+    char year_formatted[SIZEOF_LONG*5/2+2]; /* formatted year for %Y/%G */
+#endif
```

Comment:
These are only used in the guarded `else if` in L1954-1983. Could you please use intermingled declarations in that scope, for the sake of readability? This is a large function; scoping such temporary variables makes for improved readability and maintainability.

Reply:
Agreed. Thanks.